modules/nf-commons/src/main/nextflow/extension/Bolts.groovy

```diff
@@ -81,13 +82,14 @@ class Bolts {
      * @param tz The timezone to be used eg. {@code UTC}. If {@code null} the current timezone is used.
      * @return The date-time formatted as a string
      */
-    static String format(Date self, String format=null, String tz=null) {
+    static String format(Date self, String format=null, String tz=null, String locale=null) {
         TimeZone zone = tz ? TimeZone.getTimeZone(tz) : null
-        getLocalDateFormat(format ?: DATETIME_FORMAT, zone).get().format(self)
+        getLocalDateFormat(format ?: DATETIME_FORMAT, zone, locale).get().format(self)
     }
 
-    static String format(OffsetDateTime self, String format) {
-        return self.format(DateTimeFormatter.ofPattern(format))
+    static String format(OffsetDateTime self, String format, String locale=null) {
+        return self.format(DateTimeFormatter.ofPattern(format)
+                .withLocale(locale ? LocaleUtils.toLocale(locale) : Locale.default))
```

Comment:
Instead of passing an optional Locale, it should not be better to specify explicitly the english one?

Reply:
Do you mean, instead to `Locale.default`use `Locale.ENGLISH` ?

Reply:
Yes